src/lib/CRSF/CRSF.cpp

```diff
@@ -363,7 +363,16 @@ void ICACHE_RAM_ATTR CRSF::sendSyncPacketToTX() // in values in us.
     uint32_t now = millis();
     if (CRSF::CRSFstate && now >= (OpenTXsyncLastSent + OpenTXsyncPacketInterval))
     {
-        uint32_t packetRate = CRSF::RequestedRCpacketInterval * 10; //convert from us to right format
+        uint32_t packetRate;
+        if (CRSF::UARTcurrentBaud == 115200 && CRSF::RequestedRCpacketInterval == 2000)
+        {
+            packetRate = 40000; //constrain to 250hz max 
```

Comment:
This should be 4000 not 40000 if you really want 250hz or the comment should be 25hz

Reply:
It's correct, the opentx units are microseconds *!10

Reply:
Damn, schooled again.
Would it be better to keep the original line pulling the value and doing the *10, then a line after checking the baudrate and packetrate and adjusting. Just for readability.